.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,78 @@
+name: "🐞 Bug report"
+description: "Report a bug if something isn't working as expected in the Elementor"
```

Comment:
```suggestion
description: "Report a bug if something isn't working as expected in Elementor"
```

Reply:
Almost all texts were copied from the current issues templates.
Anyway, fixed :D

Reply:
Yeah I know, it just hurt my eyes 😝